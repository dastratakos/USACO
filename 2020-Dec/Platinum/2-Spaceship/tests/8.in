60 59 60
111111001110011110010101111100101111011001100111110101011001
111110001011011111110101111000111110111110111011011011110111
111111110110111001011000111101011011111101110001100101001111
110111001111011011011001110110110011111111111111101011011111
010110110100001010010111111010011011111011100011010011101111
111010110010110110101111010110100011010110111001111100111111
011111010000111111111110101101110110010110111011011111111111
111010001111011000110101011100111111011111100110011101011001
011110111011101110110101101111111111101111100100111001110110
011111101011111110111001111110101111010111111111101110101000
010111111101110101111001110100101110110101101010001010101010
110100010101111111111001011100001011101001111111101100011111
110110111111101111001011111101110101111100111101111101111111
010100100101101111111101100101111110011100111110110111011111
101001111100110011100111011011111111011011111111111011111001
111000011101010111111010011100111110111101110000101111111000
111111011111111111100111110111011110011011111110101111101000
101110001010101101000101101101101111001101101111101011111101
011011111110110111111101011101000100111010111010001111111110
110111000100111111110110001100011001111111111101110000111111
111111100001111010011001011101100100010111111001111111111111
100111111010111110110101111110011111101101010110000111100011
101110111001110001101110011001111111011011010011001000111011
111001111111110110000001110111101101111101111110011101110111
101110111110110001101011011111111001110110111011010111111110
110000011101010111101010010111000111101011011111111110001011
111111101011101111111111100110111011010011111011011100011111
101010111001010110001110001111111010010111101111111111111110
111101011111101100001010011101000111100011011010110111101111
100111101111001110010111101011111010010101001111101111111010
111111111111111011011111101010010000111011001110111101001011
111001011111101101111101010111111010100100001010101100100110
011111111111110101011111110111101010111010010011110111000010
001011111110010111101111011110111111010110110110111100110100
011111001001111001111101011111111111111010100110110011100111
101001011101010010111111010101010100010110101111111111101001
101011111111011100110111111110000101101110111011000101100101
100110111011111101111110011111111111111110111000111011011110
011100101111110101111100110111101110111011101101110111111111
111111110101110111110110101011001001000010100011011011101011
111011010101011111111001101111110111101111111100111000110011
001100001111001101110111001111011101110101001111110010110001
100111111111010110110111111111000101011110111111011011111111
101110110011111001011010110000111111111101101100001001111011
101011100011111010110101011100101011111101101011111100010100
111111111101101111011101111110001110010111111101111010111101
111101011001010110111101111011111111111001101100010011110111
101110000011101101101011010100011011100101001011111101111010
101111110101101111111111011111001011111001111111011111000100
111101001010101111111000101110110011001000110011011111011001
111111001101011111100110001100010100111011011100110110110101
111111001001010110010111001111111111110100011111111111111111
001011110111101110000111111111111111100110001001110101111110
111100001111110111110011110010010011110110111110011110011101
011110010010001001100110011101010110111111100000010100110000
011100000111101101101111110001001011111111101011101110110010
011100111101101110001111111011011101011111100100110101111010
111101110101111100001011001111111011111111011100100101111000
000001111111010011111111101111110001001001111110111111111101
111010011110111111101101000100011011111110000110111111100101
58 4 59 20
58 18 59 56
58 32 59 50
58 46 59 2
58 28 59 34
58 39 59 36
58 46 59 8
58 8 59 32
58 55 59 43
58 53 59 35
58 59 59 2
58 32 59 1
58 24 59 5
58 50 59 35
58 43 59 52
58 9 59 47
58 20 59 52
58 3 59 36
58 38 59 8
58 36 59 47
58 8 59 1
58 58 59 15
58 59 59 21
58 7 59 21
58 23 59 2
58 37 59 12
58 57 59 57
58 3 59 25
58 38 59 60
58 42 59 57
58 51 59 25
58 26 59 20
58 1 59 21
58 46 59 30
58 12 59 34
58 9 59 36
58 24 59 10
58 38 59 43
58 27 59 9
58 60 59 8
58 33 59 28
58 25 59 38
58 42 59 43
58 47 59 3
58 28 59 33
58 60 59 34
58 2 59 37
58 18 59 36
58 8 59 11
58 37 59 25
58 52 59 44
58 45 59 19
58 50 59 48
58 54 59 10
58 12 59 55
58 2 59 5
58 58 59 59
58 46 59 49
58 56 59 11
58 47 59 25
