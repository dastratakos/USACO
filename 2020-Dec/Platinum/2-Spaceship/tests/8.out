889097789
47838751
862208036
817545179
823972313
776422758
531934602
980310349
990314191
246909436
916729686
420711428
283758988
673517382
131971013
592065254
231910225
853637462
943623931
858751005
682780602
257252779
422220505
28358535
683004948
709045773
577929618
819090264
269083130
139189220
811906048
145274360
999195027
159855862
41298911
496162035
326741441
466994568
142172002
833649129
211339270
637599851
127525919
62448103
526358899
922928754
605052324
338678714
926151287
333135553
338342162
825352477
592814389
440765922
535461207
485675348
450430365
168257057
894261711
811691746
